frontend/src/AlertConfiguration/AlertConfigurationTests/AlertConfigurationAppTest.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
```

Comment:
is this still something we need to test since you're doing the alert config instead of the learn react?

Reply:
You can ignore the testing files. They are not up to date.

Reply:
You can ignore the testing files. They are not up to date.

Reply:
I think for the testing file if not relevant to the PR you can just exclude them from the PR :)